openmcyclus/DepleteReactor.py

```diff
@@ -414,7 +485,8 @@ def load(self):
         Record the number of assemblies that are to be loaded, then move 
         them from the fresh fuel inventory to the core inventory. 
         '''
-        n = min(self.n_assem_core-self.core.count, self.fresh_fuel.count)
+        n = min((self.n_assem_core-self.core.count), self.fresh_fuel.count)
```

Comment:
Are the extra set of parentheses required?

Reply:
No, it just makes it easier to read for me.